scripts/build.py

```diff
@@ -149,24 +149,34 @@
 _PARSER.add_argument(
```

Comment:
A bit unrelated to your PR, but what does this arg mean? What is terser? Maybe add a description/docstring?